README.md

```diff
@@ -1,22 +1,34 @@
 
-**Cadence** is a web radio API server. It prepares and communicates data between independent stream software _Icecast_, audio client _Liquidsoap_, and a _Postgres_ database, allowing radio administrators to broadcast audio on the filesystem to the internet while allowing users to search for music and make song requests through a custom web frontend.
+To configure Cadence, set a target directory containing your music, set a few service passwords and hostnames, and you're good to go! Cadence has container releases for amd64 and armv7. All components are containerized, and a Compose file sets the entire radio stack up in minutes. See a live demo on [cadenceradio.com](https://cadenceradio.com/)!
 
-In the current iteration (4.x) of Cadence, no frontend framework is employed and the backend is written in Go. For questions and comments, you are welcome to [open a discussion](https://github.com/kenellorando/cadence/discussions) on Github.
+![cadence5 architecture](https://user-images.githubusercontent.com/17265041/185465196-66fc2249-e43a-46f7-a12f-dbde9aaf8172.png)
 
-Check it out in action: [https://cadenceradio.com/](https://cadenceradio.com/).
+## Get Started
 
-## Feature Architecture
+> **Warning**: The way this repo currently handles configuration involves adding passwords to files which you may accidentally commit, so be careful.
 
-![cadence arch](https://user-images.githubusercontent.com/17265041/164131724-00f9a1a3-59bd-4301-947b-56cb526d3f5b.png)
+1. Edit the `cadence/config/cadence.env` file:
+   1. Set the `CSERVER_MUSIC_DIR` value to an absolute path of a directory on your local system which you want to play music. The target is not recursively searched. The default location is `/music/`.
+   2. Set the `CSERVER_REQRATELIMIT` to the number of seconds you wish to timeout users after they make song requests. Setting this value to `0` will disable rate limiting.
+2. Configure the `cadence_icecast2/config/cadence.xml` file.
+   1. Change all instances of `hackme` to a new password.
+   2. Set the `<hostname>` value to the endpoint you expect your audience to connect to. Cadence uses this value to set the stream source in the UI. This can be a DNS name, an IP address, or default `localhost` (for running locally).
+3. Configure the `cadence_liquidsoap/config/cadence.liq` file.
+   1. Change all instances of `hackme` to a new password.
+4. `docker compose up`
 
-## Discord Bot
+Running `docker compose up` will start all Cadence services. The UI is accessible by default at `localhost:8080`.
 
-_[CadenceBot](https://github.com/za419/CadenceBot/issues)_, maintained by [Ryan Hodin](https://github.com/za419), is an configurable Discord interface to Cadence API servers. The bot accepts commands through Discord text channels, and relays music into voice channels!
```

Comment:
Wow, savage.

Reply:
nice

Reply:
She'll need 5.0 support 😉 

I also suggest we make it easier for others to install their own bot instances. Containers maybe? Or package it for use for one of those "Discord Bot" distribution sites, then let server owners "one-click" install it.

Reply:
She will need 5.0 support first. That said I was also going to suggest updating the project so it becomes easy for others to install their own instances. I was thinking of a couple options: 1. containerize (since someone running now-containerized Cadence should be able to run the bot with no difficulty) or 2. package the bot however you like and give it to one of those "one-click" Discord Bot installer sites.